### PR TITLE
Remove anniversary logo and replace with best website logo

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Replaces the 200 anniversary logo with the "website of the year" logo.

Removes legacy anniversary switch and css.

**THIS SHOULD NOT BE MERGED UNTIL MAY 2022**

## Does this change need to be reproduced in dotcom-rendering ?

- [ ] No
- [x] Yes (please indicate your plans for DCR Implementation)

https://github.com/guardian/dotcom-rendering/pull/4690

## Screenshots

| Before      | After      |
|-------------|------------|
| ![before][] | ![after][] |

[before]: https://user-images.githubusercontent.com/20416599/165595712-17e24c5a-7f43-43c0-9b58-2efc01ed7a8e.png
[after]: https://user-images.githubusercontent.com/20416599/165593921-31fc4d23-5751-4cf0-b649-efbac6f4c048.png

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/main/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Comment by abeddow91** [comment]
[Remove '200 anniversary' logo](https://trello.com/c/2uLWoWyk/325-remove-200-anniversary-logo)

**Comment by jamesgorrie** [comment]
Part of: https://github.com/guardian/dotcom-rendering/issues/4536

**Review by alinaboghiu — APPROVED** [review]

**Review by alinaboghiu — CHANGES_REQUESTED** [review]
The tablet break point svg is too large and overlaps the menu

File: common/app/views/fragments/header.scala.html [hunk 1]
```diff
@@ -19,9 +19,7 @@
             aria-label="Guardian sections">
 
             <a href="@LinkTo {/}"
-            class="@{if(AnniversaryLogoHeader.isSwitchedOn) {"new-header__logo news-header__logo-anniversary"} else {
```
Comment (oliverlloyd) [inline common/app/views/fragments/header.scala.html:22]: We're removing `news-header__logo-anniversary` here but the css is still extant in the file. I see the [original PR](https://github.com/guardian/frontend/pull/23748) that added this css had a specific purpose so we should be careful to respect that (this is more a note to self at this point!)

**Comment by AshCorr** [comment]
Believe this was done in https://github.com/guardian/frontend/pull/24990 , can this be closed?

**Pull request closed without merging**